src/convert.js

```diff
@@ -15,10 +11,10 @@ const JSON_OUTPUT = `${OUTPUT_DIR}/swagger.json`
 const convertOpenApi = async function () {
   const [openapiDef] = await Promise.all([
     SwaggerParser.validate(YAML_INPUT, { dereference: { circular: false } }),
-    makeDir(OUTPUT_DIR),
+    mkdir(OUTPUT_DIR, { recursive: true }, () => {}),
```

Comment:
The promise version doesn't take a third argument for the callback ([docs](https://nodejs.org/api/fs.html#fspromisesmkdirpath-options))
```suggestion
    mkdir(OUTPUT_DIR, { recursive: true }),
```